plugins/de.cau.cs.kieler.klighd/src/de/cau/cs/kieler/klighd/filtering/GreaterThanConnective.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * KIELER - Kiel Integrated Environment for Layout Eclipse RichClient
+ *
+ * http://rtsys.informatik.uni-kiel.de/kieler
+ * 
+ * Copyright 2022 by
+ * + Kiel University
+ *   + Department of Computer Science
+ *     + Real-Time and Embedded Systems Group
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package de.cau.cs.kieler.klighd.filtering;
+
+/**
+ * A GreaterThan Connective takes one rule R and evaluates to true
+ * iff
+ * R.num > correspondingTag.num.
+ * @author tik
+ *
+ */
+public class GreaterThanConnective extends UnaryConnective {
```

Comment:
this (and some other connectives) have no `toString()` method yet, any specific reason?

Reply:
toString is defined on the abstract superclasses